subsys/bluetooth/host/hci_core.c

```diff
@@ -1929,6 +1929,20 @@ static void le_conn_update_complete(struct net_buf *buf)
 			conn->le.latency = sys_le16_to_cpu(evt->latency);
 			conn->le.timeout = sys_le16_to_cpu(evt->supv_timeout);
 
+			if (!IN_RANGE(conn->le.interval, BT_HCI_LE_INTERVAL_MIN,
+				      BT_HCI_LE_INTERVAL_MAX)) {
+				LOG_WRN("interval exceeds the valid range 0x%04x",
+					conn->le.interval);
+			}
+			if (conn->le.latency > BT_HCI_LE_PERIPHERAL_LATENCY_MAX) {
+				LOG_WRN("latency exceeds the valid range 0x%04x", conn->le.latency);
+			}
+			if (!IN_RANGE(conn->le.timeout, BT_HCI_LE_SUPERVISON_TIMEOUT_MIN,
+				      BT_HCI_LE_SUPERVISON_TIMEOUT_MAX)) {
+				LOG_WRN("supv_timeout exceeds the valid range 0x%04x",
+					conn->le.timeout);
+			}
+
```

Comment:
Missing handling of `CONFIG_BT_CONN_PARAM_ANY`

Reply:
Good point. These checks should only be applied if `CONFIG_BT_CONN_PARAM_ANY=n`